layouts/index.html

```diff
@@ -7,21 +7,23 @@
 
 		<title>OpenFisca - {{ .Title}}</title>
 
+		<link rel="stylesheet" type="text/css" href="/css/header.css">
 		<link rel="stylesheet" type="text/css" href="/css/main.css">
-		<link rel="stylesheet" type="text/css" href="/css/index.css">
 	</head>
 	<body>
 		<header class="main-header">
-			{{ partial "button-language.html" . }}
+			{{ partial "menu-links.html" . }}
 
-			<img id="logo-large" src="/img/logo-openfisca.svg" alt="OpenFisca">
+			<div class="hero">
+				<img id="logo-large" src="/img/logo-openfisca.svg" alt="OpenFisca">
 
-			<h1>{{ .Title}}</h1>
+				<h1>{{ .Title}}</h1>
 
-			<a href="{{ "countries" | relLangURL }}" class="cta get-started">{{ $.Param "get_started" }}  🙌</a>
+				<a href="{{ "countries" | relLangURL }}" class="cta get-started">{{ $.Param "get_started" }}  🙌</a>
+			</div>
 		</header>
 
-		<section class="row">
+		<section class="row" id="products">
```

Comment:
> Add a title for the products section like other pages? A link leading to a section without introduction might seem disturbing.

Reply:
Good idea, how should we name it ?
`Les produits qui utilisent OpenFisca` ?
`Powered by Openfisca` ?

Reply:
`Produits de la communauté OpenFisca` / `OpenFisca community products`? @Anna-Livia?
`Powered by OpenFisca` is great but I don't know how to translate it (`Propulsé par OpenFisca` 😅)?

Reply:
Team validated:
FR: `Propulsé par OpenFisca` 🚀 
EN: `Powered by OpenFisca` 🚀

Reply:
Done !